docs/jsonrpc/workflow.md

```diff
@@ -0,0 +1,71 @@
+# Typical workflows
+
+<!---
+Reference for mermaid
+https://mermaid-js.github.io/mermaid/#/sequenceDiagram
+-->
+
```

Comment:
Maybe some additional workflows should be included?

Reply:
I am thinking on how to monitor transaction... but it might changed in the future.